src/Conference/ConferenceItem.test.jsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import { MemoryRouter } from "react-router-dom";
+import ConferenceItem from "./ConferenceItem";
+
+test("displays conference item", async () => {
+  const mockConference = {
+    id: 1,
+    name: "Mock Conference",
+    topic: "Mock Topic",
+    dateTime: "2020-11-12T12:34:11Z",
+    city: "Mock City"
+  };
+
+  render(
+    <MemoryRouter initialEntries={[{ pathname: "/", key: "testKey" }]}>
+      <ConferenceItem conference={mockConference} />
+    </MemoryRouter>
+  );
+
+  expect(screen.getByRole("header").textContent).toBe(mockConference.name);
+  expect(screen.getByTestId("topic").textContent).toBe(mockConference.topic);
+  expect(screen.getByText("/", { exact: false }).textContent).toBe(
```

Comment:
why dont you use a data attribute here?

Reply:
Are you referring to the `.toBe("12/11/2020")`? I'm using that because if I used the `dateTime` from `mockConference` I'd have to put it through the same custom `getDatestring(new Date(dateTime))` as the component does.

Reply:
Although actually that function does have its own dedicated function so actually it might be okay to use it in the test?

Reply:
No sorry I meant `expect(getByTestid("date")` as its much more readible you can still use the getByText For the link

Reply:
I'll make it `expect(getByTestid("date")`, but I'll leave the time one how it is as I wanted to demonstrate the `exact: false` functionality.